test/Microsoft.Azure.WebJobs.EventHubs.UnitTests/EventHubConfigurationTests.cs

```diff
@@ -30,6 +30,31 @@ public EventHubConfigurationTests()
             _loggerFactory.AddProvider(_loggerProvider);
         }
 
+        [Fact]
+        public void ConfigureOptions_AppliesValuesCorrectly()
+        {
+            string extensionPath = "AzureWebJobs:Extensions:EventHubs";
+            var values = new Dictionary<string, string>
+            {
+                { $"{extensionPath}:MaxBatchSize", "123" },
+                { $"{extensionPath}:ReceiveTimeout", "00:00:33" },
+                { $"{extensionPath}:EnableReceiverRuntimeMetric", "true" },
+                { $"{extensionPath}:PrefetchCount", "123" },
+                { $"{extensionPath}:InvokeProcessorAfterReceiveTimeout", "true" },
```

Comment:
FYI - I just realized that we aren't exposing all our config properly. I just logged https://github.com/Azure/azure-webjobs-sdk/issues/1857.